src/vma/sock/sock-redirect.cpp

```diff
@@ -1792,10 +1643,6 @@ void vma_epoll_create(int epfd, int size)
 extern "C"
 int epoll_create(int __size)
 {
-	BULLSEYE_EXCLUDE_BLOCK_START
-	if (!orig_os_api.epoll_create)	get_orig_funcs();
-	BULLSEYE_EXCLUDE_BLOCK_END
-
 	do_global_ctors();
```

Comment:
do_global_ctors id called in every entry point to make all the resource allocation lazy.
i think that's a better approach in vma and changing this might cause issues with applications that the parent doesn't for any network activity and calls fork. with your change the child will fail due to the fork issue we have.